lms/static/js/certificates/views/certificate_whitelist.js

```diff
@@ -32,7 +32,12 @@
                 render: function(){
                     var template = this.loadTemplate('certificate-white-list');
                     this.$el.html(template({certificates: this.collection.models}));
-
+                    if(this.collection.isEmpty()){
```

Comment:
Per convention [1][2], there should be a space added:
- after `if`
- before `{`

[1] https://google.github.io/styleguide/javascriptguide.xml#Code_formatting
[2] https://github.com/edx/edx-platform/wiki/Javascript-standards-for-the-edx-platform